modules/gcp_helpers/authorization/main.tf

```diff
@@ -1,5 +1,6 @@
 data "google_projects" "all" {
-  filter = "parent.id=${var.organization}"
+  filter = var.project_label_filter != null ? "parent.id:'${var.organization}' labels.${var.project_label_filter.key}:${var.project_label_filter.value}" : "parent.id:'${var.organization}'"
```

Comment:
What is there are multiple keys and values in the var.project_label_filter ?

Reply:
The current definition of the `project_label_filter` variable only allows for a single key-value pair.
Would it be best to support multiple key value pairs ?